build-version.sh

```diff
@@ -30,6 +30,20 @@ if [ "$TARGET_ARCH" == "x86_64-unknown-linux-musl" ]; then
 
     CARGO_BIN_DIR="${TEMPDIR}/cargo/bin"
     CRATES2_JSON_PATH="${TEMPDIR}/cargo/.crates2.json"
+elif [ "$TARGET_ARCH" == "aarch64-unknown-linux-gnu" ]; then
+    wget $(curl https://ziglang.org/download/index.json | jq 'to_entries | map([.key, .value])[1][1]["x86_64-linux"] | .tarball' | sed -e 's/^"//' -e 's/"$//') -O zig
```

Comment:
The error from the [`shellcheck`](https://github.com/cargo-bins/cargo-quickinstall/actions/runs/3915594296/jobs/6693920222) indicates that you need to quote this to prevent word splitting:

```suggestion
    wget "$(curl https://ziglang.org/download/index.json | jq 'to_entries | map([.key, .value])[1][1]["x86_64-linux"] | .tarball' | sed -e 's/^"//' -e 's/"$//')" -O zig
```

Reply:
Note that I use `-q` here to disable the progress bar, which takes up a lot of space in the CI log, makes the debugging harder.